internal/gengapic/genrest.go

```diff
@@ -937,7 +937,7 @@ func (g *generator) lroRESTCall(servName string, m *descriptor.MethodDescriptorP
 	p("  return nil, e")
 	p("}")
 	p("")
-	override := g.getOperationPathOverride()
+	override := g.getOperationPathOverride(g.descInfo.ParentFile[serv].GetPackage())
```

Comment:
I just realized in reviewing this is that we can use the `m *descriptor.MethodDecriptorProto` here instead of `serv`, since every proto element is tied back to its parent file descriptor for this exact case. So...we don't need to add a `serv` param, and we can instead just use the `m` MethodDescriptor here.

Reply:
Fixed, thanks!